dbt/models/iasworld_test/schema.yml

```diff
@@ -0,0 +1,122 @@
+sources:
+  - name: iasworld_test
+    loaded_at_field: date_parse(loaded_at, '%Y-%m-%d %H:%i:%S.%f')
+    tags:
+      - load_auto
+      - type_res
+
+    tables:
+      - name: addn
+        description: |
+          Test environment version of the production
+          [`addn` table](#!/source/source.ccao_data_athena.iasworld.addn)
```

Comment:
Hacky internal linking as suggested [by the docs](https://docs.getdbt.com/reference/resource-properties/description#link-to-another-model-in-a-description).